spec/spatial/coordinates_proxy_spec.cr

```diff
@@ -192,6 +192,13 @@ describe Chem::Spatial::CoordinatesProxy do
     end
   end
 
+  describe ".rdgyr" do 
+    it "computes the rdgyr in-place" do 
+      s = Chem::Structure.read spec_file("FAD_can_prep.pdb")
+      s.coords.rdgyr().should be_close 6.339020, 1e-6
```

Comment:
Avoid empty parenthesis. Use default Crystal formatter to avoid this issues.

```suggestion
      s.coords.rdgyr.should be_close 6.339020, 1e-6
```